core/server/server.js

```diff
@@ -28,7 +28,7 @@ const requiredUrl = optionalUrl.required()
 
 const publicConfigSchema = Joi.object({
   bind: {
-    port: Joi.number().port(),
+    port: Joi.alternatives().try(Joi.number().port(), Joi.string()),
```

Comment:
```suggestion
    port: Joi.alternatives().try(
      Joi.number().port(),
      Joi.string().pattern(/^\\\\\.\\pipe\\.+$/),
    ),
```
Would something like this do the trick?

Reply:
Ok I've tested and applied the change.